src/commands/sourcemaps/inject.rs

```diff
@@ -23,6 +23,7 @@ pub fn make_command(command: Command) -> Command {
             Arg::new("paths")
                 .value_name("PATHS")
                 .num_args(1..)
+                .required(true)
```

Comment:
I thought that num_args does that implicitly... :(

Reply:
Yup, same.